File: boat-sample/src/test/java/sample/java/xyz/srclab/core/jvm/JvmSample.java
package sample.java.xyz.srclab.core.jvm;

import org.testng.annotations.Test;
import xyz.srclab.common.jvm.Jvms;
import xyz.srclab.common.test.TestLogger;

public class JvmSample {

    private static final TestLogger logger = TestLogger.DEFAULT;

    @Test
    public void testJvms() {
        String jvmDescriptor = Jvms.jvmDescriptor(int.class);
        //I
        logger.log("jvmDescriptor: {}", jvmDescriptor);
    }
}

File: boat-sample/src/test/kotlin/sample/kotlin/xyz/srclab/core/jvm/JvmSample.kt
package sample.kotlin.xyz.srclab.core.jvm

import org.testng.annotations.Test
import xyz.srclab.common.jvm.jvmDescriptor

class JvmSample {

    @Test
    fun testJvms() {
        val jvmDescriptor = Int::class.javaPrimitiveType!!.jvmDescriptor
        //I
        logger.log("jvmDescriptor: {}", jvmDescriptor)
    }

    companion object {
        private val logger = TestLogger.DEFAULT
    }
}